mantidproject/mantid
# Moving the status from a fit to the top of the dialog

The fit status was appearing at the bottom of the fit dialog, some users complained that they could not see the bottom of the dialog as it was off the bottom of their screen (so did not see the error messages). They also asked for more clear status reports than those provided by the GSL library (https://www.gnu.org/software/gsl/manual/html_node/Minimization-Algorithms-using-Derivatives.html)

**To test:**
1. Open muon analysis and load MUSR 62260
2. Close muon analysis
3. Click the plot and then open the fitting dialog (click the graph icon)
4. Add a polynomial of order 1. 
5. Do the fit. 
6. The status should be above the fit button and should display an error message in red
7. Increase it to a 3rd order polynomial
8. Do another fit and a different error should appear. 




Fixes #21279. 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
